smib/webserver/__main__.py

```diff
@@ -86,25 +117,21 @@ async def smib_event_handler(request: Request, event: str):
     return to_starlette_response(bolt_response)
 
 
-@router.get('/', response_class=HTMLResponse)
```

Comment:
Removed the /smib dashboard endpoint - will be revisited in another issue if and when required

Reply:
OK